opm/material/common/MathToolbox.hpp

```diff
@@ -203,6 +203,72 @@ struct MathToolbox
     { return std::pow(base, exp); }
 };
 
+// these are convenience functions for not having to type MathToolbox<Scalar>::foo()
+template <class Evaluation, class Scalar>
+Evaluation constant(const Scalar& value)
+{ return Opm::MathToolbox<Evaluation>::createConstant(value); }
+
+template <class Evaluation, class Scalar>
+Evaluation variable(const Scalar& value, int idx)
+{ return Opm::MathToolbox<Evaluation>::createVariable(value, idx); }
+
+template <class Evaluation1, class Evaluation2>
+Evaluation1 max(const Evaluation1& arg1, const Evaluation2& arg2)
+{ return Opm::MathToolbox<Evaluation1>::max(arg1, arg2); }
+
+template <class Evaluation1, class Evaluation2>
+Evaluation1 min(const Evaluation1& arg1, const Evaluation2& arg2)
+{ return Opm::MathToolbox<Evaluation1>::max(arg1, arg2); }
```

Comment:
Clearly wrong to use max() here. I think that gives the point about actually testing the values some weight...?